bundles/binding/org.openhab.binding.netatmo/src/main/java/org/openhab/binding/netatmo/internal/weather/GetStationsDataResponse.java

```diff
@@ -599,13 +599,13 @@ public int getRfLevel() {
             int level = this.rfStatus.intValue();
             int result;
 
-            if (level <= RF_STATUS_FULL_SIGNAL) {
+            if (level < RF_STATUS_FULL_SIGNAL) {
```

Comment:
[This page](https://dev.netatmo.com/dev/resources/technical/reference/weatherstation/getstationsdata) says that 60 is the highest status (lowest value) for `rf_status`, but if that's true, then no level will result in 5.  Have you observed numerically lower values than 60, meaning the Netatmo API docs are incorrect?